test/parallel/test-cli-node-print-help.js

```diff
@@ -11,9 +11,18 @@ const { exec, spawn } = require('child_process');
 const { once } = require('events');
 let stdOut;
 
+// The execPath might contain chars that should be escaped in a shell context.
+// On non-Windows, we can pass the path via the env; `"` is not a valid char on
+// Windows, so we can simply pass the path.
+const execNode = (args, callback) => exec(
```

Comment:
Should this be moved to common?

Reply:
I don't think so, (almost) each test has a different implementation.